### ISSUE TITLE
A bug that does not change the range of inferenced image when using model_ensemble.py

### ISSUE THREAD

**Issue by cksdlakstp12** [open-mmlab/mmsegmentation#2269]
Using model_ensemble.py, I found a problem by introducing the test dataset.

The images stored in the output directory were not close to gt; they were just images of black background.

When I checked, it seems that it was a problem caused by the logits not properly changing from 0 to 1 to 0 to 255 after taking argmax.

Therefore, I fixed the bug by adding a simple line of code as below.

```
        result_logits = 0
        for logit in result:
            result_logits += logit

        pred = result_logits.argmax(axis=1).squeeze()
        pred = pred * 255 # Convert an image from [0,1] to [0,255]
        img_info = dataset.img_infos[batch_indices[0]]
        file_name = os.path.join(
            tmpdir, img_info['ann']['seg_map'].split(os.path.sep)[-1])
        Image.fromarray(pred.astype(np.uint8)).save(file_name)
        prog_bar.update()
```

I hope it helps.

**Comment by MengzhangLI** [comment]
Hi, thanks for your feedback.

Could you please create a pr to fix this error? We really want to cooperate with community members to make this repo better together.

**Comment by xiexinch** [comment]
Hi @cksdlakstp12,
The `model_ensemble.py` is used to produce probability maps, it is not used to produce painted images.
I think the original implementation has no problem.

**Comment by xiexinch** [comment]
Closing the issue, as there is no activity for a while.
We hope your issue has been resolved. 
If not, please feel free to open a new one.
